src/test/kotlin/ua/marchenko/artauction/artwork/controller/ArtworkControllerTest.kt

```diff
@@ -37,9 +46,9 @@ class ArtworkControllerTest {
     }
 
     @Test
-    fun `getAllArtworks should return an empty list if there are no artworks`() {
+    fun `should return an empty list when there are no artworks`() {
         // GIVEN
-        whenever(mockArtworkService.getAll()) doReturn (listOf())
+        every { mockArtworkService.getAll() } returns (listOf())
```

Comment:
```suggestion
        every { mockArtworkService.getAll() } returns emptyList()
```
emptyList has more descriptive name

Reply:
done